tracer/src/Datadog.Trace/Debugger/Snapshots/Redaction.cs

```diff
@@ -266,7 +277,11 @@ internal static bool IsRedactedKeyword(string name)
                 return false;
             }
 
-            return !TryNormalize(name, out var result) || RedactKeywords.Contains(result);
+            var redactedKeyword = _redactedKeywordsCache.GetOrAdd(
+                name,
+                n => !TryNormalize(n, out var result) || RedactKeywords.Contains(result));
```

Comment:
The lambda here is also heap allocation, can we save that in one way or another? e.g querying the cache first and then do a `GetOrAdd` or in any other fashion

Reply:
Fixed in b8175dca2f084c5ae66180c771f92d3af73f63f8